packages/compass-workspaces/src/provider.tsx

```diff
@@ -150,10 +150,20 @@ export const WorkspacesServiceProvider: React.FunctionComponent<{
         );
       },
       openCollectionWorkspace: (namespace, options) => {
+        const isAggregationsSubtab = Boolean(
+          options?.initialAggregation ||
+            options?.initialPipeline ||
+            options?.initialPipelineText
+        );
```

Comment:
I don't think it's great that we're spreading the logic of opening a tab in multiple palces, I'd have a preference for these re-exported methods to be very thin wrappers over the openWorkspace action. We should have a single place that we can trust to always open the tab consistently across the application and inside this plugin